edit-post/components/sidebar/post-status/index.js

```diff
@@ -27,14 +27,20 @@ const PANEL_NAME = 'post-status';
 function PostStatus( { isOpened, onTogglePanel } ) {
 	return (
 		<PanelBody className="edit-post-post-status" title={ __( 'Status & Visibility' ) } opened={ isOpened } onToggle={ onTogglePanel }>
-			<PostVisibility />
-			<PostSchedule />
-			<PostFormat />
-			<PostSticky />
-			<PostPendingStatus />
-			<PostAuthor />
-			<PluginPostStatusInfo.Slot />
-			<PostTrash />
+			<PluginPostStatusInfo.Slot>
```

Comment:
It may make sense to rename the slot fill to PostStatusInfo as the slot is not only used to Plugins.

Reply:
Yes, we discussed that before with @mtias, but decided to keep the prefix `Plugin` to better express the intent. The idea is to use those fills only with `registerPlugin` API.